src/main/java/net/sf/jabref/logic/autocompleter/AbstractAutoCompleter.java

```diff
@@ -58,9 +58,12 @@ public AbstractAutoCompleter(AutoCompletePreferences preferences) {
      * Otherwise the completion is case insensitive.
      */
     @Override
-    public String[] complete(String toComplete) {
+    public List<String> complete(String toComplete) {
+        if(toComplete == null) {
+            return new ArrayList<>();
```

Comment:
If you need an empty list which is never modified but only read, you can use Collections.emptyList() which should be faster.